### PR TITLE
Add scripts to rule them all

### PR DESCRIPTION
**Pull request by pezholio**
This updates the scripts to use the [Scripts to rule them all pattern](https://github.com/github/scripts-to-rule-them-all) to make development more easy and frictionless

### CODE REVIEW

File: docker-compose.env.example [hunk 1]
```diff
@@ -1,4 +1,4 @@
-DJANGO_SECRET_KEY=
+DJANGO_SECRET_KEY=&_hnt6o2phv^vrr(%!4wo#wh2fh!0t-7=3!)8tr4so@y#d37yy
```
Comment (rjw1) [inline docker-compose.env.example]: Should this not be `SECRET_KEY_GOES_HERE` so that scripts/setup works

**Review by rjw1 — CHANGES_REQUESTED** [review]

File: script/setup [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+secret=$(script/manpy shell -c 'from django.core.management import utils; import re; print(re.escape(utils.get_random_secret_key()))' | tail -1)
```
Comment (rjw1) [inline script/setup]: I'd make this `SECRET`

File: script/setup [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
```
Comment (rjw1) [inline script/setup]: id add `-u` and `-o pipefail`

File: docker-compose.env.example (hunk 1 above)
Comment (pezholio) [reply docker-compose.env.example]: Ah, good spot 👍

**Review by rjw1 — APPROVED** [review]

**Pull request merged**